apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -358,13 +411,7 @@ public override Task<int> ExecuteNonQueryAsync(CancellationToken cancellationTok
                 };
                 mutations.Add(new Mutation {Delete = w});
             }
-
-            // Make the request. This will commit immediately or not depending on whether a transaction was explicitly created.
-            await GetSpannerTransaction().ExecuteMutationsAsync(mutations, cancellationToken)
-                .WithTimeout(TimeSpan.FromSeconds(CommandTimeout), "The timeout of the SpannerCommand was exceeded.")
-                .ConfigureAwait(false);
-            // Return the number of records affected.
-            return mutations.Count;
+            return mutations;
```

Comment:
Off-PR note: if you need to build key-sets a bit like this in multiple places, let's separate that out. (Aside from anything else, we could add `ListValue.FromValues` and `KeySet.FromKeys` methods, which would reduce the nesting by two...)